src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const content = document.querySelectorAll('.population');
+let total = 0;
+let avarage = 0;
+const populationArray = [];
+
+for (const child of content) {
+  const re = /,/gi;
+
+  populationArray.push(+child.textContent.replace(re, ''));
+};
```

Comment:
would be great to move converting logic to a separate function

Reply:
or we can find a much shorter solution
![image](https://user-images.githubusercontent.com/98764641/211297503-5703226e-dbaa-44c7-ad11-227b534d5c01.png)